Marlin/configuration_store.cpp

```diff
@@ -36,10 +36,10 @@
  *
  */
 
-#define EEPROM_VERSION "V23"
+#define EEPROM_VERSION "V24"
```

Comment:
Should this be really stored on the EEPROM ?
Despite all the philosophical discussion about this feature it's not that important to change & remember the setting value at runtime, it's more of thing to be "set and forget" at compile time IMO.